src/components/MainPage.js

```diff
@@ -55,17 +63,20 @@ class MainPage extends Component {
 			If there is an error, error message is shown with the state updated to reflect this.
 	*/
   handleExchangeSearch = (crypto, intl) => {
+  	this.setState({ loading: true });
```

Comment:
This turns on the `LoadingSpinner` and message to tell the user their data is being fetched once they've clicked the Track button.